keccak-hash/benches/keccak_256.rs

```diff
@@ -14,37 +14,36 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
-#![feature(test)]
-
-extern crate test;
-
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
 use keccak_hash::keccak;
-use test::Bencher;
 
-#[bench]
-fn bench_keccak_256_with_empty_input(b: &mut Bencher) {
+criterion_group!(keccak_256, keccak_256_with_empty_input, keccak_256_with_typical_input, keccak_256_with_large_input,);
+criterion_main!(keccak_256);
+
+pub fn keccak_256_with_empty_input(c: &mut Criterion) {
 	let empty = [0u8; 0];
-	b.bytes = empty.len() as u64;
-	b.iter(|| {
-		let _out = keccak(empty);
-	})
+	c.bench_function("keccak_256_with_empty_input", |b| {
+		b.iter(|| {
+			let _out = keccak(black_box(empty));
+		})
+	});
 }
 
-#[bench]
-fn bench_keccak_256_with_typical_input(b: &mut Bencher) {
+pub fn keccak_256_with_typical_input(c: &mut Criterion) {
 	let data: Vec<u8> = From::from("some medum length string with important information");
```

Comment:
```suggestion
	let data: Vec<u8> = From::from("some medium length string with important information");
```

Reply:
51 bytes… I have no idea what the median input size is, but it'd be nice to know.